src/styles/main.scss

```
@@ -0,0 +1,67 @@
+$clock-size: 500px;
+$second-hand-height: $clock-size * 0.4;
+$minute-hand-height: $clock-size * 0.3;
+$hand-weight: $clock-size * 0.01;
+$margin-border-dotted: $clock-size * 0.03;
+$border-dotted: $clock-size - 22px - $margin-border-dotted * 2;
+
+.page {
+  background-color: wheat;
+}
+
+.stop-watch {
+  box-sizing: border-box;
+  position: relative;
+
+  margin: 100px auto;
+  background-color: #ddd;
+  border: 10px solid #008;
```

don't forget to create variables for all colors

```suggestion
  border: 10px solid $c-black;
```